### PR TITLE
Feat: create token repository for identity service

### PR DESCRIPTION
**Pull request by Nhsdkk**
1) Create token repository
2) Edit docstring style
3) Add type annotations on __init__ methods

### CODE REVIEW

File: identity_service/db/redis_client.py [hunk 1]
```diff
@@ -15,11 +15,12 @@ class RedisClient:
     ----------
     db : redis.Redis
         Redis client
+
     """
 
     db: Redis
```
Comment (MaxPlays35) [inline identity_service/db/redis_client.py:29]: Why not async?
[click](https://redis-py.readthedocs.io/en/stable/examples/asyncio_examples.html)

**Review by MaxPlays35 — CHANGES_REQUESTED** [review]

File: identity_service/repository/token_repository.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        Parameters
+        ----------
+        user_id : str
+            User's id
+
+        Returns
+        -------
+        str
+            User's refresh token
+
```
Comment (ZixMai) [inline identity_service/repository/token_repository.py:68]: add raises to docstrings

**Review by ZixMai — CHANGES_REQUESTED** [review]
(-_-)

File: identity_service/db/redis_client.py (hunk 1 above)
Comment (Nhsdkk) [reply identity_service/db/redis_client.py:29]: fixed

File: identity_service/repository/token_repository.py (hunk 2 above)
Comment (Nhsdkk) [reply identity_service/repository/token_repository.py:68]: fixed

**Review by MaxPlays35 — APPROVED** [review]
Approved

**Review by ZixMai — APPROVED** [review]
Your docstring meme is approved!

**Pull request merged**